README.md

```diff
@@ -20,7 +20,13 @@ All the good stuff is in **./app**. Start at **./app/main.cjsx**
 
 While editing, do your best to follow style and architecture conventions already used by the project, unless you have a reason not to. **If in doubt, ask.**
 
-## Development with Docker
+#### What to do if it doesn't run
+
+Try `rm -rf ./node_modules && npm install` to freshen up your dependencies. And read the warnings, they should should tell you if you're using the wrong version of Node or npm or if you're missing any dependencies.
```

Comment:
I use `docker-compose build` instead of `npm install`, to avoid worrying about Node versions. If they have Docker installed, it's worth pointing them to the Docker section here. Or moving this section so it's closer to the Docker instructions, and suggesting they try Docker if they've got it.

Reply:
feel free to push those changes to this PR too.

Reply:
Done and done, if you want to check over my changes.